drivers/net/ethernet/renesas/rswitch.c

```diff
@@ -4299,6 +4299,54 @@ static void rswitch_fwd_init(struct rswitch_private *priv)
 		   priv->addr + FWCMPTC);
 }
 
+static void rswitch_set_max_hash_collisions(struct rswitch_private *priv)
+{
+	/* 320Mhz */
+	const u64 clk_freq = 320 * 1000 * 1000;
```

Comment:
How you obtained this frequency? Shouldn't you use `clk` subsystem for this?

Reply:
The configuration of frequency comes from ptp clock driver as a constant define for the register. So I decided to move my constant to more appropriate place.